OrbitGl/TimeGraph.cpp

```diff
@@ -456,7 +456,7 @@ const TextBox* TimeGraph::FindPreviousFunctionCall(
       for (uint64_t i = 0; i < block.size(); i++) {
         TextBox& box = block[i];
         auto box_time = box.GetTimer().m_End;
-        if ((box.GetTimer().m_FunctionAddress == function_address) &&
+        if ((box.GetTimer().m_FunctionAddress == function_address) && (!thread_ID || thread_ID.value() == box.GetTimer().m_TID) &&
```

Comment:
Nit: This should be formatted (line is too long). Did you run clang-format?

Reply:
I don't know what happened, probably I forgot it